src/bash/report_campaigns/aggregate_campaign_tier

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Parse date argument
+date=20170228
+
+if [[ -n $1 ]]
+then
+    date=$1
+fi
+
+date_length=${#date}
+if [[ $date_length != 8 ]]
+then
+    echo 'Invalid date. Example: 20170228'
+    exit
+fi
+
+echo 'Aggregating for date: '$date
+
+# Results will be put here
+location=/cms/users/$USER/campaign_tier
+hdir=hdfs://$location
+
+# Remove previous data first
+hadoop fs -rm -r $location
+
+PYTHONPATH=$(pwd)/../../python ../../../bin/run_spark /reports/aggregate_campaign_tier.py --fout=$hdir --yarn --verbose --date=$date
```

Comment:
You should fix all relative paths and never use them. They implies certain directory layout and will easily break when this is changed which will make hard to maintain the scripts. Instead you should find out how to provide full path to script, e.g. see example here:
https://github.com/vkuznet/CMSSpark/blob/master/bin/cron4phedex#L40-L47